web/packages/teleterm/src/ui/Tabs/Tabs.tsx

```diff
@@ -69,7 +69,19 @@ export function Tabs(props: Props) {
     : $emptyTab;
 
   return (
-    <StyledTabs as="nav" typography="h5" bold {...styledProps}>
+    <StyledTabs
+      as="nav"
+      typography="h5"
+      bold
+      {...styledProps}
+      css={`
```

Comment:
Any reason these new properties are defined within a `css` prop instead of being defined on `StyledTabs` directly?

Reply:
Ah no reason, fixed.